src/Skeleton/PostCreateProject.php

```diff
@@ -378,7 +378,7 @@ private static function reconfigureApplication(Event $event): void
         $offset = $matches[0][1] + mb_strlen($matches[0][0]);
         $insert = [
             '    prefix:',
-            '        locale: \'%locale%\'',
+            '        en: \'\'',
```

Comment:
Skeleton default is nl, see https://github.com/sumocoders/application-skeleton/blob/master/src/Skeleton/PostCreateProject.php#L350

Reply:
@dreadnip e181c1809cf52cdd07306237633cf1324f5632e8